components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/SubscriptionsApiService.java

```diff
@@ -1,24 +1,8 @@
 package org.wso2.carbon.apimgt.rest.api.publisher;
 
-import org.wso2.carbon.apimgt.rest.api.publisher.*;
-import org.wso2.carbon.apimgt.rest.api.publisher.dto.*;
-
-import org.wso2.msf4j.formparam.FormDataParam;
```

Comment:
These are autogenerate code. SubscriptionDTO object was changed and these are no longer used.